src/client.js

```diff
@@ -72,12 +72,12 @@ const Client = class Client {
 
   _handleMessage(topic, message) {
     var parts = topic.split('/');
-    if (parts[3] !== 'activations' && parts[3] !== 'up') {
+    if (parts[4] !== 'activations' && parts[3] !== 'up') {
       return;
     }
     var devId = parts[2];
     var payload = JSON.parse(message.toString());
-    if (parts.length === 4) { // inject app_id and dev_id, but not for field topics
+    if (parts[3] !== 'up' || parts.length === 4) { // inject app_id and dev_id, but not for field topics
       payload.app_id = this.appId;
       payload.dev_id = devId;
     }
```

Comment:
* The Node-RED node needs to output a message object on which you can set a `payload` (decoded and converted fields or binary), and a `devId`. So the message is `{ devId: '...', payload: { ... } }`. This follows Node-RED conventions
* Users listen for messages, but we can always use a callback or an event which we can decorate with the device ID as well